### PR TITLE
Reduce indentation when loading config, avoid skipping save of config if the initial config doesn't exist yet.

### PR DESCRIPTION
**Pull request by dgavedissian**

### CODE REVIEW

File: script/Systems/Files/InitFiles.lua [hunk 1]
```diff
@@ -217,26 +218,23 @@ function InitFiles:writeUserInits()
     local configPath = filepath .. filename .. filetype
     local backupConfigPath = filepath .. filename .. "_backup" .. filetype
 
-    -- Open the configuration file for reading
+    -- If a config file exists, open it so we can take a backup.
```
Comment (dgavedissian) [inline script/Systems/Files/InitFiles.lua:221]: This is the real fix

File: script/Systems/Files/InitFiles.lua [hunk 2]
```diff
@@ -217,26 +218,23 @@ function InitFiles:writeUserInits()
...
-    local oldContent = openedFileReadable:read("*a")
-    openedFileReadable:close()
+    if openedFileReadable then
+        -- Create a safety backup
+        local oldContent = openedFileReadable:read("*a")
+        openedFileReadable:close()
+
+        local backupFile, backupFileErr = io.open(backupConfigPath, "w")
+        if not backupFile then
+            Log.Warn("Cannot open backup configuration file for writing: %s", backupFileErr)
```
Comment (dgavedissian) [inline script/Systems/Files/InitFiles.lua:230]: Question for you @IllustrisJack: Should we be skipping saving of the config if writing the backup fails? Or should we just skip taking a backup?

Comment (IllustrisJack) [reply script/Systems/Files/InitFiles.lua:230]: hm good question. i´d assume if the backup fails you´d also fail in writing the config, so a general fail would make sense.

**Review by IllustrisJack — APPROVED** [review]

File: script/Systems/Files/InitFiles.lua (hunk 2 above)
Comment (dgavedissian) [reply script/Systems/Files/InitFiles.lua:230]: Fair enough

**Pull request merged**